cli/src/merge_tools/builtin.rs

```diff
@@ -844,9 +852,11 @@ mod tests {
         let no_changes_tree_id = apply_diff(store, &left_tree, &right_tree, &changed_files, &files);
         let no_changes_tree = store.get_root_tree(&no_changes_tree_id).unwrap();
         assert_eq!(
-            no_changes_tree.id(),
             left_tree.id(),
-            "no-changes tree was different",
+            no_changes_tree.id(),
+            "no-changes tree was different:\nexpected tree:\n{}\nactual tree:\n{}",
```

Comment:
comment (non-blocking): Might be worth extracting this assertion into a helper function/macro? But don't need to do this now.